src/app.types.tsx

```diff
@@ -32,36 +32,30 @@ export interface CatalogueCategory {
   is_leaf: boolean;
   catalogue_item_properties?: CatalogueCategoryFormData[];
 }
-export interface AddManufacturer {
-  name: string;
-  url?: string;
-  address: Address | undefined;
-  telephone?: string;
-}
```

Comment:
Ah sorry I meant remove AddManufacturerResponse, not AddManufacturer, github showed both the line I selected and several lines above it so it looked like I meant remove this.